### DISCUSSION TITLE
Vue 常见优化手段

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#2] [General]
# Vue 常见优化手段

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

http://localhost:4321/posts/Vue%E7%9A%84%E5%B8%B8%E8%A7%81%E4%BC%98%E5%8C%96%E6%89%8B%E6%AE%B5

**Comment by ztkuaikuai** [comment]
### 性能优化应该在遇到性能问题时针对性的优化，切忌提早优化！
